### ISSUE TITLE
WCAG 4.1.2 test 26: false positives

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#134]
[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7736437/test-document-6.pdf) fails **WCAG 4.2.1 test 26** ("_Table is incorrectly tagged as a list, or a list has invalid numbering_"). However, the error points to two entries of a list that clearly looks like a list (not a table) and also appears to be tagged correctly. It's also curious the algorithm raises this error only for the two first items of the list, but not the subsequent items. Can you check what is going on here?

**Comment by bdoubrov** [comment]
This particular error is reported correctly: two list items (one at the end of the page and another at the beginning of the next page) are tagged as a single list item. 

It is just the error message that is a bit ambiguous. We might need to disambiguate between different reasons of this error and provide a more fine-tuned error info.

**Comment by bdoubrov** [comment]
The error WCAG 4.2.1 test 26 ("Table is incorrectly tagged as a list, or a list has invalid numbering") is no longer reported. The only reported error "Invalid List item element" correctly points to two different list items split between the pages, which are incorrectly combined into a single list item in the structure tree
